opm/simulators/timestepping/AdaptiveSimulatorTimer.cpp

```diff
@@ -115,6 +115,12 @@ AdaptiveSimulatorTimer& AdaptiveSimulatorTimer::operator++ ()
         return dt_;
     }
 
+    void AdaptiveSimulatorTimer::setCurrentStepLength(double dt)
+    {
+        assert(dt > 0);
```

Comment:
this dt can be user provided i believe?

Reply:
the user is the simulator/common code here. The calling site should actually make sure that dt is appropriate, see e.g. [ScheduleState.cpp#L385](https://github.com/OPM/opm-common/blob/e4b5665cd875171da1affda36136f0e32e4a9f0a/opm/input/eclipse/Schedule/ScheduleState.cpp#L385).